Gopkg.toml

```diff
@@ -54,7 +55,11 @@ required = [
 
 [[constraint]]
   name = "github.com/coreos/etcd"
-  version = "=3.2.0"
+  version = "=3.3.9"
+
+[[constraint]]
+  name = "github.com/coreos/bbolt"
```

Comment:
When did we switch to bbolt and why?

Reply:
Something else was pulling bbolt and I had to add this constraint, otherwise a `dep ensure -update` resulted in a tree which failed to compile.

Reply:
Could you share the failure error? We use [Bolt](https://github.com/boltdb/bolt) and not [BBolt](https://github.com/coreos/bbolt) so I'm really curious what was *pulling* this dependency. There is [no mention of bbolt](https://github.com/ligato/cn-infra/search?q=bbolt&unscoped_q=bbolt) anywhere in cn-infra.

Reply:
I just pulled this out and re-ran `dep ensure -update`, and dep completed. So now I'm confused. I'l push a commit to remove it explicitly from Gopkg.toml. However, note that bbolt continues to remain in vendor, I believe because etcd is using it:

```
machine:cn-infra userid$ grep -R bbolt vendor/
vendor//github.com/boltdb/bolt/README.md:> you look at the CoreOS fork called [bbolt](https://github.com/coreos/bbolt).
grep: vendor//github.com/hashicorp/go-rootcerts/test-fixtures/capath-with-symlinks/securetrust.pem: No such file or directory
grep: vendor//github.com/hashicorp/go-rootcerts/test-fixtures/capath-with-symlinks/thawte.pem: No such file or directory
grep: vendor//github.com/coreos/etcd/cmd/tools: No such file or directory
grep: vendor//github.com/coreos/etcd/cmd/etcdctl: No such file or directory
grep: warning: vendor//github.com/coreos/etcd/cmd/etcd: recursive directory loop
grep: vendor//github.com/coreos/etcd/cmd/functional: No such file or directory
grep: vendor//github.com/coreos/etcd/Documentation/README.md: No such file or directory
vendor//github.com/coreos/etcd/mvcc/backend/backend.go:	bolt "github.com/coreos/bbolt"
vendor//github.com/coreos/etcd/mvcc/backend/batch_tx.go:	bolt "github.com/coreos/bbolt"
vendor//github.com/coreos/etcd/mvcc/backend/config_windows.go:import bolt "github.com/coreos/bbolt"
vendor//github.com/coreos/etcd/mvcc/backend/read_tx.go:	bolt "github.com/coreos/bbolt"
vendor//github.com/coreos/etcd/mvcc/backend/config_default.go:import bolt "github.com/coreos/bbolt"
vendor//github.com/coreos/etcd/mvcc/backend/config_linux.go:	bolt "github.com/coreos/bbolt"
machine:cn-infra userid$ 
```

Reply:
The error probably wasn't occurring during dep, but during running tests. Just like it's happening in Travis now. 
I believe that the reason is that etcd project uses glide and might have issue figuring out bbolt dependency.

You selected etcd v3.3.9 which [uses some specific option](https://github.com/coreos/etcd/blob/v3.3.9/mvcc/backend/config_linux.go#L31) from bbolt [introduced in bbolt v1.3.1](https://github.com/coreos/bbolt/blob/v1.3.1-coreos.6/db.go#L1003), although the dep chose v1.3.0 causing tests to fail to compile.

This means you should actually keep the bbolt in `Gopkg.toml`. However use `[[override]]` since we don't actually use bbolt directly, only as transitive dependency from etcd.

Reply:
👍